composer/airflow_1_samples/airflow_db_cleanup.py

```diff
@@ -79,9 +84,10 @@
 DAG_OWNER_NAME = "operations"
 # List of email address to send email alerts to if this job fails
 ALERT_EMAIL_ADDRESSES = []
+# Airflow version used by the environment
+AIRFLOW_VERSION = os.getenv("MAJOR_VERSION", "2.2.0").split(".")
```

Comment:
If it is for Airflow 2 why file is located in airflow_1_samples?

Reply:
Done.